Makefile

```diff
@@ -23,4 +23,7 @@ clean:
 	$(GOCMD) clean
 	rm -rf $(BUILD_DIR)
 
+install-tools:
+	go install github.com/a-h/templ/cmd/templ@latest
```

Comment:
Would prefer a versioned string here, just so we don't get a backwards compatibility issue in case they release a new major breaking version.  The following should download the latest current templ:

```shell
go install github.com/a-h/templ/cmd/templ@v0.2.543
```

Reply:
great point, added the change